src/Core/Components/Dialog/Services/IDialogInstance.cs

```diff
@@ -14,6 +16,11 @@ public interface IDialogInstance
     /// </summary>
     internal Type ComponentType { get; }
 
+    /// <summary>
+    /// Holds a reference to the previously focused element (before the dialog was opened).
+    /// </summary>
+    IJSObjectReference? PreviouslyFocusedElement { get; set; }
```

Comment:
I think this property should be `internal`.
I don't think it's a good idea to return an `IJSObjectReference` to the developer.